modules/end-user-guide/partials/proc_provisioning-jetbrains-activation-code-for-offline-use.adoc

```diff
@@ -1,55 +1,68 @@
 [id="provisioning-jetbrains-activation-code-for-offline-use_{context}"]
-= Provisioning JetBrains activation code for offline use
+= Provisioning the JetBrains offline activation code
 
-An activation code for offline use is a file with a license code that can be retrieved from the license management section of your JetBrains account for the license that is assigned to you. When you purchase a personal subscription or are assigned a commercial subscription by your organization, you receive an email prompting you to create a JetBrains account that becomes connected with the license.
-
-NOTE: When using an activation code to activate a product, you need to generate a new activation code and apply it to your product each time the subscription is renewed.
+Some editions of JetBrains IDEs require a paid subscription beyond the evaluation period, which means buying a license from JetBrains. To register a license, you need to provision to Che the JetBrains activation code for offline usage. When you renew your subscription, you will need to generate a new activation code and provision the new offline activation code.
 
 .Prerequisites
 
-* JetBrains account
-* Personal or organization subscription
+* An active link:https://account.jetbrains.com/login[JetBrains account].
+* An active link:https://www.jetbrains.com/store/comparison.html[subscription for organizations or for personal use].
+* A `base64` conversion tool.
+* An image containing the editor. See xref:building-images-for-ides-based-on-the-intellij-platform.adoc[].
+* A workspace running with the editor. See xref:using-intellij-based-ides.adoc[].
 
 .Procedure
 
-. Retrieve the activation code from the JetBrains account:
-+
-image::jetbrains/jetbrains-account.jpg[]
-+
-JetBrains provides a zip archive with two types of the activation code. Use the `<License{nbsp}ID>{nbsp}-{nbsp}for{nbsp}2018.1{nbsp}or{nbsp}later.txt` file:
-+
-image::jetbrains/activation-code.jpg[]
+. Generate the activation code by logging in to your link:https://account.jetbrains.com/login[JetBrains Account] and click the *Download activation code for offline usage* link.
 
-. Provision the activation code for offline use with {prod-short}. This procedure is performed through the {orch-name} Secrets.
+. Save `<License{nbsp}ID>{nbsp}-{nbsp}for{nbsp}2018.1{nbsp}or{nbsp}later.txt` from the downloaded zip archive as `jetbrains_license.txt`.
+
+. Run the `base64` command on `jetbrains_license.txt` to convert the activation code to base64 encoding for use as __<base64_encoded_activation_code>__ in the next step.
```

Comment:
```suggestion
. Run the `base64` command on `jetbrains_license.txt` to convert the activation code to base64 encoding for use as `__<base64_encoded_activation_code>__` in the next step.
```

Reply:
done. 
Use openssl base64 as it has a consistent implementation on various platforms. (as opposed to the  `base64` tool)